### PR TITLE
feat: Use Broker ID during initial client registration

### PR DESCRIPTION
**Pull request by mladkau**
This PR adds the Broker ID to the initial connection URL when a client connects to the server. This change is required for load-balancing requests to the server.

### CODE REVIEW

File: lib/client/socket.js [hunk 1]
```diff
@@ -26,6 +19,15 @@ module.exports = ({ url, token, filters, config, identifyingMetadata }) => {
     throw error;
   }
 
+  const Socket = Primus.createSocket({
+    transformer: 'engine.io',
+    parser: 'EJSON',
+    plugin: {
+      'emitter': require('primus-emitter'),
+    },
+    pathname : '/primus/'+token,
```
Comment (mladkau) [inline lib/client/socket.js]: In the moment this defaults to just /primus (see: https://github.com/primus/primus/blob/master/index.js#L67) but the server will accept anything under /primus. The token can be picked up by a load balancer to route the request. See: https://github.com/snyk/ops/blob/master/helm/broker-container-server/templates/load-balancing.yaml#L13

Comment (JackuB) [reply lib/client/socket.js]: What are we going to load balance? Isn't it stateless thus current round-robin should be enough? Maybe I'm missing some context for this PR.

Comment (mladkau) [reply lib/client/socket.js]: The broker is not stateless as Agent and Broker Client need to access the same server instance to be able to communicate.

Comment (JackuB) [reply lib/client/socket.js]:
```suggestion
    pathname: `/primus/${token}`,
```

**Pull request merged**